### DISCUSSION TITLE
Identification cards and other items cannot be hidden. Items hidden in the ventilation can be easily sold. Was this intended?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15002] [Ideas and suggestions]
Identification cards hidden in the ventilation are visible to the captain in the navigator terminal under the "hull" tab, as if a crew member is located now in that room.
Items hidden in the ventilation can be found in the navigator terminal under the "search" tab.
This makes the existence of secret places almost pointless, as their location can be easily discovered by anyone looking for the hidden item.

Items hidden in the ventilation can be easily sold.
IMHO these item must be hidden from merchant as if the item in anyone inventory

**Comment by Ek-Videogames** [comment]
Tbh the 'hidden' containers should probably inherit the "nosell" & "notake" tags by default.

Also maybe having hidden containers' contents not show up on the item finder too.
